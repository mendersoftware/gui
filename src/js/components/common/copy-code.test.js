import React from 'react';
import renderer from 'react-test-renderer';
import CopyCode from './copy-code';

describe('CopyCode Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<CopyCode code="sudo it all!" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
