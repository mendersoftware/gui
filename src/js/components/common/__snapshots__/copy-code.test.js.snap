// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CopyCode Component renders correctly 1`] = `
Array [
  <div
    className="code"
  >
    <button
      className="MuiButtonBase-root MuiIconButton-root"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      style={
        Object {
          "float": "right",
          "margin": "-20px 0 0 10px",
        }
      }
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiIconButton-label"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M16 1H4c-1.1 0-2 .9-2 2v14h2V3h12V1zm-1 4l6 6v10c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h7zm-1 7h5.5L14 6.5V12z"
          />
        </svg>
      </span>
    </button>
    <span
      style={
        Object {
          "whiteSpace": "pre-wrap",
          "wordBreak": "break-word",
        }
      }
    >
      sudo it all!
    </span>
  </div>,
  <p />,
]
`;
